algorithms/2-search-solutions/fibonnaci_recursive.rb

```diff
@@ -0,0 +1,36 @@
+require 'benchmark'
+
+def fib(n)
+   if (n == 0)
+     # #1
+     return 0
+   elsif (n == 1)
+     # #2
+     return 1
+   else
+     # #3
+     return fib(n-1) + fib(n-2)
+   end
```

Comment:
Not that I would recommend it, but I just noticed that you could easily write this like:

    def fib(n)
      n < 2 ? n : fib(n - 1) + fib(n - 2)
    end

Also, you should put spaces around your minuses.